server/routes/user/refreshToken.route.js

```diff
@@ -0,0 +1,30 @@
+var UserModel = require('../../models/user.model');
+var PasswordHasher = require('../../helper/passwordHasher');
+const jwt = require('jsonwebtoken');
+
+var refreshToken = function(req, res) {
+  var token = req.body.token || req.query.token || req.headers['x-access-token'] || req.headers['authorization'];
+  if (!token) {
+    return res.status(401).json({message: 'Invalid Token'});
+  }
+  jwt.verify(token, process.env.JWT_SECRET, function(err, user) {
+    if (err) throw err;
+    UserModel.findById({
+      '_id': user._id
+    }, function(err, usr) {
+      if (err) throw err;
```

Comment:
This error is unhandled?

Reply:
Good catch, I’ll return 500 instead. Thanks